io/net/htx_nic_devices.py.data/htx_nic_devices.yaml

```diff
@@ -11,4 +11,4 @@ peer_ips: ""
 net_ids: ""
 # time limit in minutes
 time_limit: 2
-htx_rpm: ""
+htx_rpm_link: "http://ausgsa.ibm.com//gsa/ausgsa/projects/h/htx/public_html/htxonly/"
```

Comment:
Do not change the parameter name keep it htx_rpm  the reason
1. htx block device uses htx_rpm as param
2. all my CR input files has htx_rpm as input, I cannot change them all.. also  if a new param gets added ..than it is duplicate one extra param

Reply:
@abdhaleegit sure, will maintain same param.